### ISSUE TITLE
Feature Request: zone redundancy information on replicas

### ISSUE THREAD

**Issue by warrior88** [microsoft/azure-container-apps#565]
**Is your feature request related to a problem? Please describe**.  
A clear and concise description of what the problem is
When zone redundancy has been enabled, there is no information on how the replicas are distributed across the zones.
 
**Describe the solution you'd like**.    
A clear and concise description of what you want to happen, any scenarios it will unblock, etc.  
There should be a table showing the replicas and the zones they have been deployed.
If there are 3 replicas, the zone column should show that the replicas are distributed across AZ 1, 2 and 3.

**Describe alternatives you've considered**.   

**Additional context**.  
The information will assure that the deployment is able to meet the SLA of 99.95% and not concentrated within a zone.

**Comment by wgkesler** [comment]
I agree that replicas distribution over the zones is an important piece of information, and should be available over the portal or Azure API.

**Comment by simonjj** [comment]
yes, this would be very valuable to have
